bentoml/_internal/server/service_app.py

```diff
@@ -12,7 +13,7 @@
 from ..server.base_app import BaseAppFactory
 from ..service.service import Service
 
-if t.TYPE_CHECKING:
+if TYPE_CHECKING:  # try not use t.TYPE_CHECKING to avoid pylint unused-import
```

Comment:
```suggestion
if TYPE_CHECKING:   # pragma: no cover # pylint: disable=unused-import 
```

Reply:
Reject. this will also omit the the imports that really not used

Reply:
ugh usually for types they will be detected as unused-import?

Reply:
No. pylint actually know if you are using `if TYPE_CHECKING`. But not that smart at lease now.

Reply:
but doing individual imports from typing is a lot slower than just `import typing as t` 🤷🏼

Reply:
You mean from `8.11us` to `10us` ?

Reply:
in my case it is some us to some ms